### PR TITLE
digitalocean should be one word.

### PR DESCRIPTION
**Pull request by gtmanfred**
### What does this PR do?

Everything on their webpage is a reference to DigitalOcean

https://en.wikipedia.org/wiki/DigitalOcean not https://en.wikipedia.org/wiki/Digital_Ocean

### Tests written?

Yes

### CODE REVIEW

**Review by cachedout — COMMENTED** [review]
I have a couple of pretty big concerns here.

It looks like the driver names are being changed in config files, which would mean that people would have to update their configs. There's no indication anywhere in release notes or anything that would happen nor is there a deprecation path.

We don't need to update `man` pages at all. Those are automatically compiled.

In short, while this seems like a worthwhile goal it needs to be transparent to users. Breaking configurations just for a spelling change isn't something we should be doing IMHO.

**Comment by gtmanfred** [comment]
This has a virtual_alias for `do` and for `digital_ocean` so the old driver name will still work, and no one who has it setup should need to change anything.

Since we have been having problems with deprecating stuff, I don't really see a need to deprecate it here, especially since virtualaliases let us make a change without breaking older configs.

https://github.com/saltstack/salt/pull/43588/files#diff-747a8b0ee0f009374043c29399673d63R63

**Comment by cachedout** [comment]
Sounds good. If you could remove the changes to the man page and also add a note in the release notes just so it's perfectly clear to people what is happening here, I think we can get this in. Thanks.

**Comment by gtmanfred** [comment]
Will do, thanks.

**Comment by gtmanfred** [comment]
Man pages removed, and release notes added.

Waiting for the one test to fail so I can fix it.

**Comment by cachedout** [comment]
Test failure is here: https://jenkins.saltstack.com/job/PR/job/salt-pr-linode-ubuntu14-n/14863/testReport/junit/unit.utils.test_schema/ConfigTestCase/test_optional_requirements_config/

**Comment by gtmanfred** [comment]
darn multiline string skipped out of my regex!

Ok, tests should be good this time.

Thanks,
Daniel

**Comment by rallytime** [comment]
Looks like that test is still failing: https://jenkins.saltstack.com/job/PR/job/salt-pr-linode-ubuntu16-py3/2082/testReport/junit/unit.utils.test_schema/ConfigTestCase/test_optional_requirements_config/
:-/

**Comment by gtmanfred** [comment]
there were more that I missed, I verified that it will run fully this time.

File: salt/cloud/clouds/digitalocean.py [hunk 1]
```diff
@@ -59,10 +59,11 @@
 # Get logging started
 log = logging.getLogger(__name__)
 
-__virtualname__ = 'digital_ocean'
+__virtualname__ = 'digitalocean'
+__virtual_aliases__ = ('digital_ocean', 'do')
```
Comment (rallytime) [inline salt/cloud/clouds/digitalocean.py]: 👍

**Review by rallytime — APPROVED** [review]

**Comment by rallytime** [comment]
Remaining test failures are unrelated.

**Pull request merged**